osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1067,6 +1067,35 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/darc.svg"
       },
       "coingecko_id": "konstellation"
+    },
+    {
+      "description": "The native token of Gravity Bridge",
+      "denom_units": [
+        {
+          "denom": "ibc/E97634A40119F1898989C2A23224ED83FDD0A57EA46B3A094E287288D1672B44",
+          "exponent": 0,
+          "aliases": [
+            "ugraviton"
+          ]
+        },
+        {
+          "denom": "graviton",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/E97634A40119F1898989C2A23224ED83FDD0A57EA46B3A094E287288D1672B44",
+      "name": "Graviton",
```

Comment:
"name" : Gravity bridge ?

Reply:
No, the name of the token is Graviton

Reply:
I believe we are in agreement no further change is necessary, hence resolving.